wallet/src/actors/app/methods.rs

```diff
@@ -771,20 +783,29 @@ impl App {
             .flatten()
             .inspect(|res| {
                 log::debug!("Periodic request result: {:?}", res);
+                let status = serde_json::from_value::<types::SyncStatus>(res.clone());
+                log::debug!("The result of the node status is {:?}", status.unwrap());
```

Comment:
Can we handle this error? If this unwrap cannot fail, please say it, otherwise lets try to handle it